src/quail/slate/SlateEndpoints.hpp

```diff
@@ -9,10 +9,10 @@
 template <typename T>
 class EndPoint : public SlateKey<T>{
     public:
-        EndPoint(const std::string id, const std::string quailID, T init, bool editable) : 
-        SlateKey<T>(id, init), quailID(quailID), editable(editable) {}
+        EndPoint(const std::string id, const std::string quailID, T init, bool editable, const std::string desc = "none") : 
```

Comment:
I am vaugely concerned about how memory for this is being allocated, but who knows I guess.